attributes/config.rb

```diff
@@ -0,0 +1,6 @@
+
+default['apache_zookeeper']['config_dir'] = '/opt/zookeeper/current/conf'
+default['apache_zookeeper']['bin_dir'] = '/opt/zookeeper/current/bin'
```

Comment:
The `install_dir` and these two can get out of sync which we create some strange behavior. I would likely prefer just using those library methods or deriving from `install_dir`.